### PR TITLE
docs: Improve qs for webui user

### PR DESCRIPTION
**Pull request by tara-det-ai**
## Closed

Starting with a new PR.

## Description

Provide instructions for the WebUI user who wants to run their first experiment.

Use case: The new WebUI user has just signed in to their Determined cluster. They don't know how to get started. They want to create an experiment. They are provided a link to a quickstart.

webui dashboard with empty project

## Checklist

- [ ] Changes have been [manually QA'd](https://determined-ai-docs.s3.us-west-2.amazonaws.com/previews/037ed4215f98a822d2c8d386b87a3f64/get-started/qs-webui.html). To refresh the link, visit the [instructions](https://hpe-aiatscale.atlassian.net/wiki/spaces/DOC/pages/1341423617/About+the+Docs+Preview+Tool).

## To Do

- [ ] Web team creates ticket to modify the hyperlink from WebUI dashboard to the new quickstart. Old URL: "https://docs.determined.ai/latest/tutorials/quickstart-mdldev.html" | New URL "https://docs.determined.ai/latest/get-started/webui-qs.html".

## Test Plan

- [ ] As Admin, host a Determined cluster and create a new user.
- [ ] Sign out.
- [ ] Launch WebUI as the new user. 
- [ ] WebUI shows no experiments and provides user with link to a quickstart.
- [ ] Click the quickstart.
- [ ] Steps are reproducible.

### CODE REVIEW

**Review by prevuri-determined — CHANGES_REQUESTED** [review]

File: docs/get-started/qs-webui.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      locally
+
+      Train a single model for a fixed number of batches, using constant values for all
+      hyperparameters on a single *slot*. A slot is a CPU or CPU computing device, which the
+      Determined master schedules to run.
+
+      #. Download and extract the tar file: :download:`mnist_pytorch.tgz
+         <../examples/mnist_pytorch.tgz>`.
+
+      #. Install the Determined library and start a cluster locally:
```
[the remark this answers is not in the document]
Comment (tara-det-ai) [reply docs/get-started/qs-webui.rst]: resolved by adding a note similar to the remote instructions and by removing the extra instructions, keeping them only in the prerequisites section

File: docs/get-started/qs-webui.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+            det experiment create distributed.yaml .
+
+         You can also use the ``-m`` option to specify a remote master IP address:
+
+         .. code:: bash
+
+            det -m http://<ipAddress>:8080 experiment create distributed.yaml .
+
+      #. In the WebUI dashboard, click the **Experiment** name to view the experiment’s trial
```
[the remark this answers is not in the document]
Comment (tara-det-ai) [reply docs/get-started/qs-webui.rst]: added

File: docs/get-started/qs-webui.rst [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+            export DET_MASTER=<ipAddress>:8080
+
+      #. Create and run the experiment:
+
+         .. code:: bash
+
+            det experiment create distributed.yaml .
+
+         You can also use the ``-m`` option to specify a remote master IP address:
```
[the remark this answers is not in the document]
Comment (tara-det-ai) [reply docs/get-started/qs-webui.rst]: resolved as discussed

File: docs/get-started/qs-webui.rst [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+         If you examine the ``distributed.yaml`` file, you'll notice the
+         ``resources.slots_per_trial`` field is set to a value of ``8``:
+
+         .. code:: yaml
+
+            resources:
+               slots_per_trial: 8
+
+         This is the number of available GPU resources. The ``slots_per_trial`` value must be
```
[the remark this answers is not in the document]
Comment (tara-det-ai) [reply docs/get-started/qs-webui.rst]: done

File: docs/get-started/qs-webui.rst [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+         .. code:: bash
+
+            det experiment create const.yaml .
+
+         The final dot (.) argument uploads all of the files in the current directory as the
+         *context directory* for your model. Determined copies the model context directory contents
+         to the trial container working directory.
+
+      #. In the WebUI dashboard, click the **Experiment** name to view the experiment’s trial
```
[the remark this answers is not in the document]
Comment (tara-det-ai) [reply docs/get-started/qs-webui.rst]: added

**Review by lbliii — APPROVED** [review]

**Pull request closed without merging**